src/screens/ImportWalletScreen.tsx

```diff
@@ -434,6 +437,11 @@ export class ImportWalletScreen extends PureComponent<Props, State> {
     });
   };
 
+  executeWithNetworkConnectionCheck = (func: () => void) => () => {
```

Comment:
Wouldn't it be better to name it `handleOnPress`?

Reply:
I think it gives more info about what is doing

Reply:
Makes sense actually.

I also propose to rename everywhere `func` to `callback` or `cb`

Reply:
done